src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj

```diff
@@ -1,15 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
   <Import Project=".\Common.GenevaExporter.props"/>
 
   <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>net8.0;$(NetStandardMinimumSupportedVersion)</TargetFrameworks>
```

Comment:
please call out this in changelog.md file, thanks!

Reply:
done